1_space_odyssey/js/camilla-9000/app.js

```diff
@@ -0,0 +1,131 @@
+        ______\/////________\////////////____________\///___________\///////////________\///////////______\///////////////_________\///________
+  
+      ============================================================================================
+        TIMELINE of EVENTS
+      ============================================================================================
+       2001: A Space Odyssey spans from the dawn of man to its title year. 
+             Stone Age Earth: 
+             In the presence of a mysterious black obelisk, pre-humans discover the use 
+             of tools -- and weapons -- violently taking first steps toward intelligence. 
+             
+        1999: On Earth's moon astronauts uncover another mysterious black obelisk. 
+        
+        2001: Between Earth and Jupiter, the spacecraft's intelligent computer makes a mistake that 
+              kills most of the human crew -- then continues to kill to hide its error. 
+              
+        2020: The sole survivor of the journey to Jupiter ascends to the next level of humanity.
+ */
+
+
+/*
+  You are the sole survivor who will make the journey to Jupiter.
+
+  But before you can leave you must build an autonomous humanoid robot to help you on your mission.
+
+  This humanoid will be used to perform dangerous and mission critical tasks on your journey to Jupiter.
+  - space walks to repair the spaceship
+  - fight off alien creatures
+  -  
+
+  The humanoid is your doppelgänger, a sentient being that will help you save man from extinction by keeping you alive.
+
+  Using JavaScript build the software that will allow us to describe and programme the humanoid to do our bidding.
+  */
+
+/* ______   __  __   ______  ______   
+        _..._
+      .'     '.      _
+     /    .-""-\   _/ \
+   .-|   /:.   |  |   |
+   |  \  |:.   /.-'-./
+   | .-'-;:__.'    =/
+   .'=  *=|Noroff_.='
+  /   _.  |    ;
+ ;-.-'|    \   |
+/   | \    _\  _\
+\__/'._;.  ==' ==\
+         \    \   |
+         /    /   /
+         /-._/-._/
+  ...    \   `\  \
+          `-._/._/
+
+
+ 1. CREATE DROID::::::
+    Below are all the variables needed to initialize an android into consciousness in its default state.
+    a. assign the correct values to the variables below, 
+    b. ensure to choose the correct values and datatypes
+    c. the droid should be built in your likeness (a human, with human features, 2 arms, legs etc)
+*/
+var isAlive = true;
+var name = "Camilla-9000";
+var hasJetPack = true;
+var suitColor = "white";
+var eyes = true;
+var eyeColor = "green";
+var hairColor = "yellow";
+var legs = 2;
+var arms = 2;
+var hands = 2;
+var ears = 2;
+var canWalk = true;
+var canRun = true;
+var canFly = true;
+var hasWeapon = false;
+var isInGoodMood = false;
+
+console.log(".-.-. Booting .-.-.");
+console.log(name, suitColor, isInGoodMood, canRun, isAlive, hasJetPack, eyes, eyeColor, hairColor, legs, arms, hands, ears, canWalk, canFly, hasWeapon);
+
+
+if(hasJetPack) {
```

Comment:
please move this code to just after the description on point 2. description. 
and 

please fix indenting.
````js
if (someThing) {
  ...
} else {
  ...
}
````

![one](https://media.giphy.com/media/ag3PWAeHrCdWV0tlkD/200w_d.gif)